web/server/codechecker_server/migrations/config/versions/126fa3f55e4b_added_permission_for_product_view.py

```diff
@@ -0,0 +1,54 @@
+"""Added permission for Product View
+Revision ID: 7ceea9232211
+Revises: 302693c76eb8
+Create Date: 2021-05-25 09:43:15.402946
+"""
+# revision identifiers, used by Alembic.
+revision = '7ceea9232211'
+down_revision = '302693c76eb8'
+branch_labels = None
+depends_on = None
+from alembic import op
+import sqlalchemy as sa
```

Comment:
Please keep the white spaces in this file similarly what I pasted here: https://github.com/Ericsson/codechecker/pull/3332#discussion_r658557838
```suggestion

from alembic import op
import sqlalchemy as sa

```

Also check the other part of this file.

Reply:
Oops Sorry! I had copied text from your suggestion, but it gets copied without formatting I guess. Nevertheless, I have fixed and rebased it